components/server/src/authorization/authorizer.ts

```diff
@@ -513,15 +509,23 @@ export class Authorizer {
     }
 }
 
-// TODO(gpl) remove after FGA rollout
-export async function isFgaAuthorizerEnabled(userId: string): Promise<boolean> {
+export async function isFgaChecksEnabled(userId: string): Promise<boolean> {
     return getExperimentsClientForBackend().getValueAsync("centralizedPermissions", false, {
         user: {
             id: userId,
         },
     });
 }
 
+export async function isFgaWritesEnabled(userId: string): Promise<boolean> {
+    const result = await getExperimentsClientForBackend().getValueAsync("spicedb_relationship_updates", false, {
+        user: {
+            id: userId,
+        },
+    });
+    return result || isFgaChecksEnabled(userId);
```

Comment:
I had to check twice here, because of the expected await. But should work fine :+1:

Reply:
Thanks, I will update it to make it clear